scripting/ttt_item_ump.sma

```diff
@@ -92,6 +93,20 @@ public cswa_damage(weapon_id, victim, attacker, Float:damage)
 	if(get_weapon_edict(weapon_id, REPL_CSWA_ITEMID) == g_iItemID)
 	{
 		new Float:angles[3];
+		new colors[3];
+		new VictimState = ttt_get_playerstate(victim);
+		if(VictimState == PC_DETECTIVE || VictimState == PC_TRAITOR || VictimState == PC_INNOCENT)
+			{
+				if(VictimState == PC_TRAITOR)
+				colors = {255,0,0};
+				if(VictimState == PC_DETECTIVE)
+				colors = {0,0,255};
+				if(VictimState == PC_INNOCENT)
+				colors = {0,255,0};
+				
+				set_user_rendering(victim,kRenderFxGlowShell,colors[0],colors[1],colors[2],kRenderNormal,25);
+				set_task( 2.5, "remove_glowump", victim );
+			}
```

Comment:
There is no need for colors variable because in ttt_const.ini file there is g_iTeamColors, so you can use it his way:
g_iTeamColors[VictimState][0] - red
g_iTeamColors[VictimState][1] - green
and so on. Also that if condition isn't necessary.

P.S. Why do you guys want to UMP to be color highlighter? :/ For that we have Golden Gun.